.github/workflows/writecache.yml

```diff
@@ -14,16 +14,19 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # ratchet:actions/checkout@v4
+      - name: Reset timestamps
+        shell: bash
+        run: git ls-files -z | xargs -0 touch -r go.mod
       - name: Init Hermit
         uses: cashapp/activate-hermit@12a728b03ad41eace0f9abaf98a035e7e8ea2318 # ratchet:cashapp/activate-hermit@v1.1.4
+      - name: Build Cache
```

Comment:
Won't this result in the cache constantly growing? i.e. this will download all artifacts from previous runs, and they will be included in the new cache entry even if they are no longer required by the build?